aviary/mission/flops_based/ode/mission_ODE.py

```diff
@@ -67,7 +67,7 @@ def setup(self):
         aviary_options = options['aviary_options']
         core_subsystems = options['core_subsystems']
         subsystem_options = options['subsystem_options']
-        num_engine_type = len(aviary_options.get_val('engine_models'))
+        engine_count = len(aviary_options.get_val(Aircraft.Engine.NUM_ENGINES))
```

Comment:
The name here relates to that whole discussion about number of hetergenous engines, etc. I guess it is best addressed in a future PR.